modules/related-posts/jetpack-related-posts.php

```diff
@@ -235,7 +258,7 @@ public function get_target_html() {
 	 *
 	 * @returns string
 	 */
-	public function get_target_html_unsupported() {
+	public function get_client_rendered_html_unsupported() {
```

Comment:
This is probably out of scope of this PR, but is Related Posts supposed to appear in the loop? The doc comment implies not, but I'm seeing them on a test homepage (both on this branch and `master`).